StreamingKit/StreamingKit/STKAudioPlayer.m

```diff
@@ -3237,6 +3257,8 @@ -(void) setMeteringEnabled:(BOOL)value
 	}
 	else
 	{
+        __weak STKAudioPlayer* weakSelf = self;
```

Comment:
@haritowa please use correct indentation for this line (i.e. move it 1 tab to the right)